galsim/table.py

```diff
@@ -20,6 +20,8 @@
 addition of the docstring and few extra features.
 """
 from . import _galsim
+import galsim
```

Comment:
It's generally not a good idea to import the base module from a submodule, since galsim in turn imports things from table.py, which can lead to circular dependencies.  But in this case, I don't see where you even use this.  So probably can just remove this line.